docs/products/postgresql/howto/migrate-aiven-db-migrate.rst

```diff
@@ -124,22 +144,16 @@ The command output should be similar to the following stating that the ``pg_dump
     done            null
 
 
-
 .. Note::
     The overall ``method`` field is left empty due to the mixed methods used to migrate each database.
 
 
-5. Remove the configuration from the destination service via :doc:`../../../tools/cli`::
+5. Remove the configuration from the destination service via :doc:`../../../tools/cli`:: Make sure your migration process is in one of the following state when triggering the removal: ``done`` for the ``pg_dump`` method, and ``syncing`` for logical replication. Otherwise, removing a migration configuration can leave the destination cluster in an inconsistent state.
```

Comment:
The command below doesn't render correctly because it needs to have that `::` double colon on the end of the paragraph, rather than in the middle.

Reply:
Fixed